frontend/src/kuberos.vue

```diff
@@ -121,6 +121,9 @@ export default {
       .get(url)
       .then(function(response) {
         _this.kubecfg = response.data;
+        if(_this.kubecfg.email == "") {
+          _this.kubecfg.email = "default-user";
```

Comment:
How about we call the user `kuberos` to make it obvious when looking at the kubecfg file where it comes from? Bonus points for making it a constant. :)

Reply:
I'm changed the default user name to kuberos, but I don't know how to work with constants in vue.